core/io/resource.cpp

```diff
@@ -528,6 +521,33 @@ String Resource::get_id_for_path(const String &p_path) const {
 #endif
 }
 
+#ifdef TOOLS_ENABLED
+Vector<ConfigurationInfo> Resource::get_configuration_info() const {
+	Vector<ConfigurationInfo> ret;
+
+	Array info;
+	if (GDVIRTUAL_CALL(_get_configuration_info, info)) {
+		ret.resize(info.size());
+
+		int idx = 0;
+		for (const Variant &variant : info) {
+			ret.set(idx++, ConfigurationInfo::from_variant(variant));
+		}
+	}
+
+	return ret;
+}
+#endif
+
+void Resource::update_configuration_info() {
+#ifdef TOOLS_ENABLED
+	if (!ConfigurationInfo::configuration_info_changed_func) {
+		return;
+	}
+	ConfigurationInfo::configuration_info_changed_func(this);
```

Comment:
```suggestion
	if (ConfigurationInfo::configuration_info_changed_func) {
		ConfigurationInfo::configuration_info_changed_func(this);
	}
```
Early return doesn't make much sense here.